designer/server/src/routes/forms/library.js

```diff
@@ -11,14 +12,39 @@ export default [
   ({
     method: 'GET',
     path: '/library',
-    async handler(request, h) {
-      const { auth } = request
-      const token = auth.credentials.token
-      const model = await library.listViewModel(token)
-
-      return h.view('forms/library', model)
-    },
     options: {
+      handler: async (request, h) => {
+        const { auth, query } = request
+        const token = auth.credentials.token
+
+        const page = Number(query.page) || 1
```

Comment:
Can we use Joi with defaults for this like we do in forms-manager?
`options.validate.query = joi.object().keys({ page: joi.number().default(1), ... })`.